tests/Infrastructure.Tests/StatusUpdates/StatusUpdateRepositoryTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Application.StatusUpdates;
+
+using Domain.StatusUpdates;
+
+using Infrastructure.Databases;
+
+using Microsoft.Extensions.Logging;
+
+namespace Infrastructure.Tests.StatusUpdates;
+
+public class StatusUpdateRepositoryTests : IClassFixture<TestDatabaseFixture>
+{
+    private readonly TestDatabaseFixture _fixture;
+    private IStatusUpdateRepository _cut;
+
+    public StatusUpdateRepositoryTests(TestDatabaseFixture fixture)
+    {
+        _fixture = fixture;
+        EFDatabaseContext context = _fixture.CreateContext();
+        _cut = new EFStatusUpdatesRepository(context, Mock.Of<ILogger<EFUserRepository>>());
+        context.Database.BeginTransaction();
+    }
+
+    [Fact]
+    public void AddStatusUpdateStoresTheStatusUpdate()
+    {
+        // Arrange
+        var statusUpdate = new StatusUpdate();
+
+        // Act
+        _cut.Add(statusUpdate);
+        IEnumerable<StatusUpdate> statusUpdates = _cut.All;
```

Comment:
nit:
It's best to keep the act-section focused on a single thing. I'd move this part up into arrange since it's part of the setup needed to complete the test:
```suggestion
        // Arrange
        var statusUpdate = new StatusUpdate();
        _cut.Add(statusUpdate);

        // Act
        IEnumerable<StatusUpdate> statusUpdates = _cut.All;
```